packages/web-components/fast-components/src/radio-group/fixtures/base.html

```diff
@@ -10,6 +10,15 @@ <h4>Defaults</h4>
         </fast-radio-group>
     </div>
 
+    <div>
+        <fast-radio-group orientation="vertical" name="Bourbon">
+            <label slot="label">Bourbon</label>
```

Comment:
Not to be a literal buzz kill, but could we change this to a non-alcoholic drink or colors or something? :)

Reply:
O'douls it is

Reply:
done, tree types